lib/screens/guidelinesPage.dart

```diff
@@ -246,9 +246,9 @@ class RootCategoriesPage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-        backgroundColor: Styles.shadowWhite,
-        body: SingleChildScrollView(
+    return Container(
+        color: Styles.shadowWhite,
+        child: SingleChildScrollView(
```

Comment:
Made Scaffold a container because `RootCategoriesPage` is a Page and not a screen. So it is already in the `Scaffold` in `navigation_bar_controller.dart`,